server/tests/utils.py

```diff
@@ -10,3 +10,21 @@ def standard_seed_db(postgresql):
             with open("tests/db-data.psql", "rt") as f:
                 cursor.execute(f.read())
                 conn.commit()
+
+
+def check_output(output: str, strings):
+    """Checks if specified output (presumably stdout) has appropriate content. strings
+    is a list of strings that are expected to be present. They're expected
+    to appear in the specified order, but there may be other things
+    printed in between them. Will assert if string is not found. Returns number
+    of found strings."""
+    offset = 0
+    cnt = 0
+    for s in strings:
+        new_offset = output.find(s, offset)
```

Comment:
If you don't use this variable then you can move right side of this assignment directly to ```if``` clause.

Reply:
Good catch! It's now being used.